scripts/prepare-code-coverage-artifact.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Concord
+#
+# Copyright (c) 2021 VMware, Inc. All Rights Reserved.
```

Comment:
Please check if copyright is same as in other files/scripts

Reply:
Copyright have been updated to 2018-2021.